### ISSUE TITLE
[Adaptive Card > OrderDelivery.Json]: Voice over does convey the incorrect information as 'Dimmed' for the 'Delivery' button. 

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7600]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.20220628.3)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 15.5
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "OrderDelivery.json" control and double tap to activate it.
7. Navigate to the 'Delivery' button and observe the issue.

**Actual Result:**
Voice over does convey the incorrect information as 'Dimmed' for the Delivery button and on double tapping on it no action is getting performed.
Also, Delivery button does not looks like a disabled control.

**Expected Result:**
Voice over should convey the correct information for the 'Delivery' button and when focus comes on it then it should announce as 'Delivery button'.

**User Experience:**
It will impact the screen reader user if the voice over does convey the incorrect information for the non-disabled control.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176617423-76c39ee6-8b85-4dcb-8c1d-75c58a155cbb.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue can be tracked using URL below, hence closing this issue here.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/50
